vulnerabilities/helpers.py

```diff
@@ -79,7 +79,7 @@ def create_etag(data_src, url, etag_key):
     return True
 
 
-is_cve = re.compile(r"CVE-\d+-\d+", re.IGNORECASE).match
+is_cve = re.compile(r"CVE-\d+-\d+$", re.IGNORECASE).match
```

Comment:
IMHO the regex `CVE-\d{4}-\d{4,7} ` is more explicit and accurate.  Let's use that instead.